testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -997,6 +998,7 @@ public void queryCompleted(QueryCompletedEvent queryCompletedEvent)
                 closeAllSuppress(e, queryRunner);
                 throw e;
             }
+            queryRunner.getCoordinator().getInstance(Key.get(ConnectorServicesProvider.class)).loadInitialCatalogs();
```

Comment:
Shouldn't we do it in `io.trino.server.testing.TestingTrinoServer#TestingTrinoServer`?

Reply:
good point